server/bin/pbench-backup-tarballs.py

```diff
@@ -158,11 +162,11 @@ def backup_to_local(
         # for other errors where we still want to backup in S3.
         return Status.FAIL
 
-    backup_controller_path = os.path.join(lb_obj.backup_dir, controller)
+    backup_controller_path = Path(lb_obj.backup_dir, controller)
 
     # make sure the controller is present in local backup directory
     try:
-        os.mkdir(backup_controller_path)
+        backup_controller_path.mkdir()
```

Comment:
Consider using the `exist_ok` parameter set to True to avoid the need for the `try/except` block.